services/QuillLMS/app/services/clever_integration/libary_student_importer.rb

```diff
@@ -0,0 +1,23 @@
+module CleverIntegration
+  class LibaryStudentImporter
```

Comment:
nbd but Library is missing an R here and in the file name!

Reply:
Good catch.  I must have been channeling my high school Latin teacher who pronounced it that way.